elections/uk/lib.py

```diff
@@ -24,38 +26,43 @@ def shorten_post_label(post_label):
     'party_ec_id',
 ]
 
-def get_extra_csv_values(person, election):
-    theyworkforyou_url = ''
+def get_extra_csv_values(person, election, post):
+    gss_code = ''
```

Comment:
Yeah, the approach to what to put in the CSV output has been pretty much "whatever *might* be helpful" :) (although ISTR we may have added this / the parlparse ID to make it easier for TheyWorkForYou to import the elected people)